e2e-tests/data/sequence-adaptation.js

```diff
@@ -0,0 +1,12 @@
+(() => {
+  globalThis.CONDITIONAL_KEYWORDS = {};
+  globalThis.LOOP_KEYWORDS = {};
+  globalThis.GLOBALS = [];
+  globalThis.LINT = function commandDictionary(commandDictionary, view, node) {
+    const diagnostics = [];
+
+    console.log(commandDictionary, view, node);
```

Comment:
is this log needed?

Reply:
Yeah, the linter fails on it otherwise. We could ignore the file if we wanted to do that instead though.

Reply:
Ah. If you don't plan on using a variable, we can prepend `_` before the variable name (i.e. `_commandDictionary`) to get around that.